### PR TITLE
update README file

### PR DESCRIPTION
**Pull request by ma2bd**
## Motivation

Improve the README file

## Proposal

* Improve text
* Simplify the example
* Add link to linera.dev

## Test Plan

CI

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,35 +6,47 @@
...
 
-6. [`linera-core`](https://linera-io.github.io/linera-protocol/linera_core/index.html) The core Linera protocol, including client and server logic, node synchronization, etc.
+* [`linera-storage`](https://linera-io.github.io/linera-protocol/linera_storage/index.html) Defines the storage abstractions for the protocol on top of `linera-chain`.
 
-7. [`linera-rpc`](https://linera-io.github.io/linera-protocol/linera_rpc/index.html) Defines the data-type for RPC messages (currently all client<->proxy<->chain<->chain interactions), and track the corresponding data schemas.
+* [`linera-core`](https://linera-io.github.io/linera-protocol/linera_core/index.html) The core Linera protocol, including client and server logic, node synchronization, etc.
 
-8. [`linera-service`](https://linera-io.github.io/linera-protocol/linera_service/index.html) Executable for clients (aka CLI wallets), proxy (aka validator frontend) and servers.
+* [`linera-rpc`](https://linera-io.github.io/linera-protocol/linera_rpc/index.html) Defines the data-type for RPC messages (currently all client<->proxy<->chain<->chain interactions), and track the corresponding data schemas.
```
Comment (afck) [inline README.md:37]: Does this work?
```suggestion
* [`linera-rpc`](https://linera-io.github.io/linera-protocol/linera_rpc/index.html) Defines the data-type for RPC messages (currently all client &#x2194; proxy &#x2194; chain &#x2194; chain interactions), and track the corresponding data schemas.
```

(At least in a GitHub comment `&#x2194;` becomes: &#x2194;.)

File: README.md [hunk 2]
```diff
@@ -43,24 +55,29 @@ export PATH="$PWD/target/debug:$PATH"
 # Import the optional helper function `linera_spawn_and_read_wallet_variables`.
 source /dev/stdin <<<"$(linera net helper 2>/dev/null)"
 
-# Run a local test network with the default parameters and 0 extra user wallets.
-# This will set environment variables LINERA_{WALLET,STORAGE}_0 referenced by -w0 below.
-linera_spawn_and_read_wallet_variables linera net up --extra-wallets 0
+# Run a local test network with the default parameters and a number of microchains
+# owned by the default wallet.
+# The helper function will set the 2 environment variables LINERA_{WALLET,STORAGE}.
```
Comment (afck) [inline README.md:60]:
```suggestion
# The helper function will set the two environment variables LINERA_{WALLET,STORAGE}.
```

File: README.md [hunk 3]
```diff
@@ -43,24 +55,29 @@ export PATH="$PWD/target/debug:$PATH"
...
+linera_spawn_and_read_wallet_variables \
+linera net up
 
 # Print the set of validators.
-linera -w0 query-validators
+linera query-validators
 
-# Query some of the chains created at genesis
+# Query the default balance (aka the "chain balance") of some of the chains.
```
Comment (afck) [inline README.md:67]: Maybe just "chain balance"? ("Default" sounds like how much a new chain would have in the beginning or something.)

**Review by afck — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -6,35 +6,47 @@
...
+
+* our [developer manual](https://linera.dev), and
+
+* our [whitepaper](https://linera.io/whitepaper).
+
 
 ## Repository Structure
 
-The Linera protocol repository is broken down into the following crates and subdirectories: (from low-level to high-level in the dependency graph)
+This repository is broken down into the following main crates and subdirectories: (from low-level to high-level in the dependency graph)
```
[the remark this answers is not in the document]
Comment (ma2bd) [reply README.md:21]: It was written by a native speaker :) but happy to change

File: README.md (hunk 1 above)
Comment (ma2bd) [reply README.md:37]: Not sure about this one. It makes the markdown hard to read

Comment (ma2bd) [reply README.md]: actually let me try

**Pull request merged**